### ISSUE TITLE
Throwing TypeError when validating an invalid schema

### ISSUE THREAD

**Issue by M-TGH** [4Catalyzer/graphql-validation-complexity#19]
**Usage info**
GraphQL: apollo-server-lambda@1.3.2
Package version: graphql-validation-complexity@0.2.2

**What is happening?**
Currently running into GraphQL not returning a correct JSON response when sending a query that is invalid based on the schema.
This also logs a warning to the graphql server logs:
```
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Cannot read property 'getCostFactor' of undefined
```

**What did you expect to happen?**
Expected graphql to return an error specifying that the query was invalid.

**Reason of error:**
In the code for the rule `validationContext.getFieldDef` is used without correctly checking on it's returned value. 
Looking at [the code for getFieldDef](https://github.com/graphql/graphql-js/blob/master/src/validation/ValidationContext.js#L210) the return type uses a maybe type, which means it could possibly be undefined or null.

**Proposed fix:**
Add some checks on whether the fieldDef value is undefined/null and handle it accordingly.

**Willing to open a PR?**
Yes, will open a PR. 👍 However not sure on the handling which should be done when fieldDef is undefined and how much effect this would have on the validation of the cost. (although cost shouldn't matter much when the query is invalid anyways)

**Comment by M-TGH** [comment]
Closing since #20 has been merged 👍
